web-app/js/portal/cart/WpsDownloadHandler.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.cart');
+
+Portal.cart.WpsDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
```

Comment:
This class is *very* similar to `GogoduckDownloadHandler`, apart from some small differences in labels, URL construction, and subset parameters.  But overall, the logic is the same.

Consider extracting out the common code (to a `Portal.cart.AsyncDownloadHandler` class, perhaps).